src/bin/tikv-importer.rs

```diff
@@ -45,11 +45,10 @@ extern crate tikv;
 extern crate toml;
 
 #[cfg(unix)]
-mod profiling;
 #[macro_use]
-mod setup;
-use setup::*;
-mod signal_handler;
+mod utils;
```

Comment:
we use `util` mostly.

Reply:
since `tikv::util` is imported here too, how about `bin_util`?

Reply:
Can we put this util into a different namespace? or can we use absolute path, like super::util?